docs/_documentation/fundamentals/view-presenters.md

```diff
@@ -25,6 +25,43 @@ Let's take a look now at the methods of that interface:
 - `ChangePresentation(MvxPresentationHint hint)`: This method is the one that gets called whenever a PresentationHint is requested. It is responsible for handling the requested change.
 - `Close(IMvxViewModel toClose)`: As you can imagine, this method is used to handle the close request of a ViewModel. It takes the ViewModel instance to be closed as parameter.
 
+## View Presenters on each platform
+Each mobile platform has its own View Presenter:
+- Android: [MvxAndroidViewPresenter](https://github.com/MvvmCross/MvvmCross/blob/develop/MvvmCross/Droid/Droid/Views/MvxAndroidViewPresenter.cs)
+- iOS: [MvxIosViewPresenter](https://github.com/MvvmCross/MvvmCross/blob/develop/MvvmCross/iOS/iOS/Views/Presenters/MvxIosViewPresenter.cs)
+- Windows: [MvxWindowsViewPresenter](https://github.com/MvvmCross/MvvmCross/blob/develop/MvvmCross/Windows/Uwp/Views/MvxWindowsViewPresenter.cs)
+
+When you navigate to selected ViewModel, platform specific View Presenter handles displaying View properly.
+
+<p align="center">
+  <img src="https://preview.ibb.co/cGuy8Q/View_Presenters.png">
```

Comment:
Thanks Marc, of course but unfortunately I do not have permission to upload it. :(